<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secrum â€” Login</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sign in to Secrum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phoneLineEdit">
     <property name="placeholderText">
      <string>Phone number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="codeLineEdit">
     <property name="placeholderText">
      <string>Verification code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sendCodeButton">
     <property name="text">
      <string>Send code</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
